### PR TITLE
F5 parser fix for irules with multiline single command lines.

### PR DESCRIPTION
**Pull request by jeffkala**
Add a preprocessing step that sanitizes the F5 configs completely removing the `ltm rule` configuration sections.  These sections are not uniform or standardized which makes a regex find or other solution very challenging.  ltm rule compliance checking is probably not frequently needed.

### CODE REVIEW

**Comment by jeffkala** [comment]
fixes #78

File: netutils/config/parser.py [hunk 1]
```diff
@@ -597,6 +597,104 @@ class F5ConfigParser(BaseBraceConfigParser):
...
+    def __init__(self, config):
+        """Create ConfigParser Object.
+
+        Args:
+            config (str): The config text to parse.
+        """
+        super(F5ConfigParser, self).__init__(self._clean_config_f5(config))
+
+    def _clean_config_f5(self, config_text):  # pylint: disable=no-self-use
+        """Removes all configuration items with 'ltm rule'.  iRules are essentially impossible to parse with the lack of uniformity.
```
Comment (jmcgill298) [inline netutils/config/parser.py:609]:
```suggestion
        """Removes all configuration items with 'ltm rule'.
        
        iRules are essentially impossible to parse with the lack of uniformity,
        therefore, this method ensures they are not included in ``self.config``.
```

**Review by jmcgill298 — CHANGES_REQUESTED** [review]

File: netutils/config/parser.py [hunk 2]
```diff
@@ -597,6 +597,130 @@ class F5ConfigParser(BaseBraceConfigParser):
...
+
+        iRules are essentially impossible to parse with the lack of uniformity,
+        therefore, this method ensures they are not included in ``self.config``.
+
+        Args:
+            config_text (str): The entire config as a string.
+
+        Returns:
+            str: The sanitized config with all iRules (ltm rule) stanzas removed.
+        """
```
Comment (qduk) [inline netutils/config/parser.py:619]: Should this function have an example? I don't know that we've defined is private functions should have an example. I just noticed this one doesn't and the one below does.

Comment (itdependsnetworks) [reply netutils/config/parser.py:619]: I'll leave it to Jeff to decide, most private methods don't have it, but it is at times helpful to understand what is expected.

**Review by itdependsnetworks — APPROVED** [review]

File: netutils/config/parser.py (hunk 2 above)
Comment (jeffkala) [reply netutils/config/parser.py:619]: I don't think it makes sense.  In most cases I wouldn't expect someone to directly interact with a private method, by putting an example on it and having it in the docs might influence that.

**Pull request merged**